ocaml/xapi/export.ml

```diff
@@ -118,15 +120,16 @@ let rec update_table ~__context ~include_snapshots ~preserve_power_state
            )
        ) ;
     (* add VTPMs that belong to this VM *)
-    vm.API.vM_VTPMs
-    |> List.iter (fun ref -> if Db.is_valid_ref __context ref then add ref) ;
+    if not (List.mem Devicetype.VTPM excluded_devices) then
```

Comment:
Do we allow users to exclude vtpms as well (and then add them back later on)?

Reply:
Yes, this code is doing the exclusion. For adding it back to the VM, I don't think there's an easy way to export the TPM contents separately